### ISSUE TITLE
Error when attempting to install FreeDOS on an HD image

### ISSUE THREAD

**Issue by blindphoneman** [joncampbell123/dosbox-x#2528]
Hi:

If I try to install FreeDOS onto a virtual HD image, it boots up, but, when setup tries to format or even install onto an already formatted image, I get the following:

CPU_HLT attempted to set HLT_Decode while CPU decoder already HLT_Decode

Any suggestions?

Thanks!

**Comment by Allofich** [comment]
That message is created in /src/cpu/cpu.cpp. Just in case it's useful for tracking down the issue, here's the commit where that line was added

https://github.com/joncampbell123/dosbox-x/commit/844047596a67a5c658aedd2a31c804612ff3c80f

**Comment by Wengier** [comment]
Try set ``core=normal`` (instead of the dynamic core) in the config file for the FreeDOS installation. I tried with FreeDOS 1.3 RC4 and seemed to work.

**Comment by Wengier** [comment]
FreeDOS 1.3 RC4 after installation:

![image](https://user-images.githubusercontent.com/8216923/118347423-46b0de80-b511-11eb-9f4d-38f484d40d84.png)

**Comment by blindphoneman** [comment]
Yes, that did the trick. Thanks!  Sent from Mail for Windows 10 From: WengierSent: Friday, May 14, 2021 11:20 PMTo: joncampbell123/dosbox-xCc: blindphoneman; AuthorSubject: Re: [joncampbell123/dosbox-x] Error when attempting to install FreeDOS on an HD image (#2528) Try set core=normal (instead of the dynamic core) in the config file for the FreeDOS installation. I tried with FreeDOS 1.3 RC4 and seemed to work.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
